app/src/frontend/building/categories.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import Sidebar from './sidebar';
+
+const Categories = (props) => (
+    <Sidebar>
+        <Category
+            title="Location"
+            slug="location"
+            help="https://pages.colouring.london/location"
+            inactive={false}
+            mode={props.mode}
+            building_id={props.building_id}
+        />
```

Comment:
Nit-pick: could just do `{...props}` in each `Category` to reduce repetition?

Reply:
I'm less keen on passing `{...props}` through since realising it makes it easy to rely on things implicitly which may be set up two or more layers away - "Explicit is better than implicit" in a Zen-of-Python kind of way.

Reply:
True. What could reduce the implicitness would be having TS interfaces defined for prop types of both components - then if a field was added to the prop types of Categories but not Category, it would result in a transpilation error, because the destructuring process would try to assign to properties that do not exist on Category.